test/shareArticle.spec.js

```diff
@@ -0,0 +1,65 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../server/app';
+
+chai.use(chaiHttp);
+const { expect } = chai;
+
+describe('Share Article Test', () => {
+  let testToken;
+  before('Login user', (done) => {
+    chai
+      .request(app)
+      .post('/api/v1/users/auth/login')
+      .send({
+        email: 'postman@gmail.com',
+        password: 'Password'
+      })
+      .end((err, res) => {
+        expect(res.body).to.be.an('object');
+        testToken = res.body.data.token;
+        done();
+      });
+  });
+
+  it('should return Invalid articleId', (done) => {
```

Comment:
This description is the same as the one on line 38, but the actual tests are different.

Reply:
Thank you, James I've made the correction.